content/calico/stars_policy_demo/apply_network_policies.md

```diff
@@ -32,13 +38,21 @@ Network policies in Kubernetes use labels to select pods, and define rules on wh
 
 Create two new network policies.
 
-##### `allow-ui.yaml`
+Copy/Paste the following commands into your Cloud9 Terminal.
+```
+cd ~/environment/calico_resources
```

Comment:
You can remove this command (seems redundant from previous command)
cd ~/environment/calico_resources

Reply:
I'm leaving it in there, because it's harmless, and I want to make sure we get the file in the right place.